masschallenge/django-accelerator
# [AC-7750] : Display all-MassChallenge programs events from Eventbrite

## Changes made to [AC-7750](https://masschallenge.atlassian.net/browse/AC-7750)
- Added all masschallenge events field to the Program

**How to test**
- Checkout to this branch as well as the sibling PR on [Accelerate](https://github.com/masschallenge/accelerate/pull/2649)
- Run migrations..
- On the admin page, check the "All Masschallenge Events" checkbox for a program
- Assign the mentor to the demoadmin
- Visit the [events](http://localhost:8181/events/) page.
- Notice that there is a checkbox for All Masschallenge events
